src/test/java/seedu/notus/util/parser/ParseListNoteCommandTest.java

```diff
@@ -2,14 +2,325 @@
 
 //@@author R-Ramana
 
+import seedu.notus.command.ArchiveNoteCommand;
+import seedu.notus.command.Command;
+import seedu.notus.command.ListNoteCommand;
+import seedu.notus.command.UnarchiveNoteCommand;
 import seedu.notus.data.exception.SystemException;
 
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
 class ParseListNoteCommandTest {
 
-    private ParseCreateOrDeleteTagCommand parser;
+    private ParseListNoteCommand parser;
     private SystemException exception;
+    private Command command;
     private String inputString;
     private String expectedMessage;
     private String actualMessage;
 
+    @Test
+    void parseCommand_validMessage_returnNewCommand() {
+        inputString = "/tag cs2113 ";
+
+        parser = new ParseListNoteCommand(inputString);
+
+        try {
+            command = parser.parse();
+            assertTrue(command instanceof ListNoteCommand);
+        } catch (SystemException exception) {
+            exception.printStackTrace();
+        }
+
+        parser = new ParseListNoteCommand(inputString);
+
+        try {
+            command = parser.parse();
+            assertTrue(command instanceof ListNoteCommand);
+        } catch (SystemException exception) {
+            exception.printStackTrace();
+        }
```

Comment:
The inputString is not changed? Likwise there are few calls but the inputString wasnt changed.